build/pipelines/feature-flag-ci.yml

```diff
@@ -7,6 +7,7 @@ pr:
   paths:
     include:
       - src/features.xml
+      - build/pipelines/feature-flag-ci.yml

```

Comment:
So! This file isn't _including_ itself in the sense that it will run yaml contained inside itself twice. This is part of the paths "included" in the trigger for this pipeline. i.e. this pipeline _should run_ when either of these files is changed!